Gemfile

```diff
@@ -1,3 +1,9 @@
 source 'https://rubygems.org'
 
+group :development, :test do
+  gem 'simplecov', require: false
+  gem 'rubocop', '~> 0.37.2', require: false unless RUBY_VERSION =~ /^1.8/
```

Comment:
Yes. It would be another status check independent from Travis.

Sent from my iPhone

> On 29. 3. 2016, at 8:50, Justin French <notifications@github.com> wrote:
> 
> In Gemfile:
> 
> > @@ -1,3 +1,9 @@
> >  source 'https://rubygems.org'
> >  
> > +group :development, :test do
> > +  gem 'simplecov', require: false
> > +  gem 'rubocop', '~> 0.37.2', require: false unless RUBY_VERSION =~ /^1.8/
> @mikz if we only run on code climate, does that mean it's added as a build status thing alongside Travis?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>